src/Illuminate/View/Component.php

```diff
@@ -147,7 +147,7 @@ public function resolveView()
                 return $view;
             }
 
-            return $this->extractBladeViewFromString($view);
+            return $this->extractBladeViewFromString(strval($view));
```

Comment:
It's probably best to not invoke the method at all.

```suggestion
            if (is_null($view)) {
                return '';
            }

            return $this->extractBladeViewFromString($view);
```

Reply:
Maybe, as `extractBladeViewFromString` only accepts strings:

```
if (! is_string($view)) {
    return '';
}
```

Reply:
Thanks!